_maps/RandomRuins/LavaRuins/lavaland_surface_legionbarracks.dmm

```diff
@@ -83,9 +83,10 @@
 "S" = (
 /obj/structure/rack,
 /obj/item/claymore{
```

Comment:
I would prefer if this was it's own item like obj/item/claymore/ruin instead of being mapedited

Reply:
Do you want me to just make separate items for Excal/baracks so they can be adjusted appropriately or given quirky special shit